src/styles/application.css

```diff
@@ -1,5 +1,13 @@
 .sto-app {
   margin: auto 1.7em;
+  height: 100vh;
```

Comment:
`100vh` is a risky move to do, especially on mobile. 
For mobile browsers, 100vh > 100%

Reply:
Fixed in the following media query rule.